src/main/java/io/nats/client/api/StorageType.java

```diff
@@ -34,15 +31,9 @@ public String toString() {
         return policy;
     }
 
-    private static final Map<String, StorageType> strEnumHash = new HashMap<>();
-
-    static {
-        for (StorageType env : StorageType.values()) {
-            strEnumHash.put(env.toString(), env);
-        }
-    }
-
     public static StorageType get(String value) {
-        return strEnumHash.get(value);
+        if (File.policy.equalsIgnoreCase(value)) { return File; }
+        if (Memory.policy.equalsIgnoreCase(value)) { return Memory; }
+        return null;
```

Comment:
Half the code, one less object. I would still use the other strategy on enums with more entries